ts/models/conversation.ts

```diff
@@ -1261,6 +1281,22 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     }
   }
 
+  public async setIsApproved(value: boolean) {
+    if (value !== this.get('isApproved')) {
+      window?.log?.info(`Setting ${this.attributes.profileName} isApproved to:: ${value}`);
+      this.set({
+        isApproved: value,
+      });
+
+      // to exclude the conversation from left pane messages list and message requests
+      if (value === false) {
+        this.set({ active_at: undefined });
+      }
+
+      await this.commit();
+    }
```

Comment:
what happens if message requests toggle is off? this conversation won't be visible anywhere?